pom.xml

```
@@ -26,6 +26,22 @@
             <version>3.27.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.java-websocket</groupId>
+            <artifactId>Java-WebSocket</artifactId>
+            <version>1.5.4</version>
```

why not use the lastest version?
```suggestion
            <version>1.6.0</version>
```

---

Oops, bad copy-paste 😔